### PR TITLE
tfsec: Add Version 0.63.0

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)

- [x] ✨ Add Manifest
- [ ] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

## Description

Add Manifest for TFSec which ia a static analysis for terraform.

note : I had to fixed the version to 0.63.0 since the windows version is not available for the last 0.63.1... So in the manifest it's missing the autoupdate and checkversion section... Once it resolved we should re-add these sections

## Related Tickets & Documents

None

### Add/Modify a description in the manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, <del>because they aren't needed</del> Because there an issue with 0.63.1 for windows, it doesn't exist 😒

### Update Hashes?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

:mag: :bug: :punch: :wave: :rotating_light:

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### tfsec

- [x] Description
- [x] License
- [x] Hashes

**Review by gbonnand — APPROVED** [review]

**Pull request merged**
